development/kops/delete_cluster.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+# Copyright 2020 Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
We will probably need to exit on any error in this script. Adding something like the following at the beginning of the script may be useful?
```
set -euo pipefail

err_report() {
    echo "Exited with error on line $1"
}
trap 'err_report $LINENO' ERR
```

Reply:
I had that in there originally, but the problem I was having was if the cluster was down, it would not clean up the s3 bucket. I think it is better to try each step kind of like `rm -rf *`

Reply:
If the cluster is down, do we want to delete the back-end bucket?

Reply:
Yes.